clang/include/clang/Basic/riscv_vector.td

```diff
@@ -2011,9 +2021,15 @@ let ManualCodegen = [{
           RVVConvBuiltinSet<"vfncvt_xu_f_w", "csi", [["Uv", "UvFwu"]]>;
       let OverloadedName = "vfncvt_f" in {
         defm :
-          RVVConvBuiltinSet<"vfncvt_f_x_w", "csi", [["Fv", "Fvwu"]]>;
+          RVVConvBuiltinSet<"vfncvt_f_x_w", "ci", [["Fv", "Fvwu"]]>;
```

Comment:
Isn't the 'c' version the one that needs Zfh? Assuming 'c' is the destination type. f16->i8

Reply:
I think we don't need to support `c` here, since we don't have `f8` supported yet.

Reply:
This is `int->f` conversion, I think the `f->int` version is already handled by current approach, isn't it?

Reply:
Oh I missed that it was int->fp.

The original crash I found was narrowing fp->int https://godbolt.org/z/Khfx8j6jW

Reply:
I'm a bit confused, the [spec](https://github.com/riscv/riscv-v-spec/blob/master/v-spec.adoc#184-zvfhmin-vector-extension-for-minimal-half-precision-floating-point) says `When the Zvfhmin extension is implemented, the vfwcvt.f.f.v and vfncvt.f.f.w instructions become defined when SEW=16. `. Does it mean we `zvfhmin` only supports those 2 version of conversion, or it means those 2 version are included?

Reply:
Zvfhmin only supports those 2 versions of conversion.

The base Zve/V spec doesn't support any FP operands with EEW=16. Zvfhmin extends the base Zve/V extensions by adding just those 2 conversions.

Reply:
So we don't need to support `fp->int` conversion in `zvfbfmin` which is in this example: https://godbolt.org/z/Khfx8j6jW right?

Reply:
Correct, it is not supported. We need to block the declaration in the frontend instead of generating a fatal error in the backend.

Reply:
Yeah, I think we are not able to handle this in current approach, I'll have another PR to use the `RequiredFeatures` and remove current type checking approach.